chia/_tests/util/test_testnet_overrides.py

```diff
@@ -9,10 +9,8 @@ def test_testnet10() -> None:
     overrides: Dict[str, Any] = {}
     update_testnet_overrides("testnet10", overrides)
     assert overrides == {
-        "SOFT_FORK2_HEIGHT": 3000000,
         "SOFT_FORK4_HEIGHT": 4465000,
         "HARD_FORK_HEIGHT": 2997292,
-        "HARD_FORK_FIX_HEIGHT": 3426000,
```

Comment:
If this will no longer work with testnet 10, should the entire function here be removed?

Reply:
yeah, that probably makes sense